content/cloud/custom-domain.md

```diff
@@ -0,0 +1,47 @@
+title = "Custom Domains"
+template = "cloud_main"
+date = "2023-07-26T12:00:00Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/custom-domain.md"
+
+---
+- [Setting Up Custom Domains](#setting-up-custom-domains)
+  - [Relationship Between Custom Domains and Spin Applications](#relationship-between-custom-domains-and-spin-applications)
+  - [Next Steps](#next-steps)
+  - [Terminology](#terminology)
+
+Every Fermyon Cloud application has a default domain name in the format `<app-name>-<random-string>.fermyon.app`. The Fermyon Cloud Custom Domains feature provides a way for you to use your own domain names; providing your users with human-friendly access to your Fermyon Cloud applications. Another advantage of using Fermyon Cloud Custom Domains is that your Spin application can have a stronger sense of branding and identity. For example, for [Finicky Whiskers: The World's Most Adorable Manual Load Generator](https://www.fermyon.com/blog/finicky-whiskers-part-1-intro) we have moved away from using the default domain name `finicky-whiskers-6cavavfh.fermyon.app`, and we much prefer using the custom domain `finickywhiskers.com`. 
+
+If you do not own a domain (and are **not** interested in purchasing one) but would like to customize your Spin application's domain name within the `fermyon.app` apex domain, then Custom Fermyon Subdomains may be a good option for you. Check out the [Custom Fermyon Subdomains tutorial](./custom-fermyon-subdomain.md) for more instructions. 
+
+If you do not own a domain and you **are** interested in purchasing one, you can purchase one through a domain registrar like [Namecheap](https://www.namecheap.com/) or [Google Domains](https://domains.google). At this time, Fermyon Cloud is not a domain registrar meaning you'll have to use an external registrar.
```

Comment:
> At this time,

Just checking to make sure: do we want to lead the reader onto the fact that we are considering this? If so, is there something we want to provide the user to ask for feedback?

Reply:
Good call Matt, if we do become a registrar we will just say we are ... then.